bliss/conf/base_config.yaml

```diff
@@ -119,13 +119,14 @@ encoder:
         ]
 
 generate:
+    process_index: 0
     n_workers_per_process: 0
     n_batches: ${simulator.n_batches}
     batch_size: ${simulator.prior.batch_size}
-    max_images_per_file: 5000
+    max_images_per_file: 4096
     cached_data_path: ${paths.data}/cached_dataset
     file_prefix: dataset
-    files_start_idx: 0
+    num_files_per_process: 128
```

Comment:
Actually, what if we removed `process_index` and `num_files_per_process` from this base config, to make it cleaner, but append `+generate.process_index` in the calls to the `bliss` executable in the parallel generate script? (See next few comments.) This abstracts away these details about multi-process data generation, since we ship this base config file but not the /scripts directory.

Also, currently `n_batches` together with `batch_size` determines the total number of images, and together with `max_images_per_file` determines the total number of files to be generated (by this process). Meaning either one of `num_files_per_process` or `n_batches` could be unnecessary.